src/components/switch/css/sass/Switch.scss

```diff
@@ -8,9 +8,9 @@ knobMargin = 0.2em;
 }
 
 .fl-switchUI-control {
-    width: knobSize * 3;
+    width: ($knobSize * 3);
     border: 0.15em solid var(--fl-fgColor, currentColor);
-    border-radius: knobSize + knobMargin;
+    border-radius: ($knobSize + $knobMargin);
```

Comment:
Looking at these changes, I wonder if we should just convert this to vanilla CSS and use CSS variables and `calc`. There isn't much nesting in this file either, so not sure how much it's actually benefitting from pre-processing.

Reply:
It's not really benefitting from being Sass on its own, but it's used as a partial import here: https://github.com/fluid-project/infusion/blob/944aa2d2ce3d2cde0cdfcf19ff0cde593a367e17/src/framework/preferences/css/sass/PrefsEditor.scss#L205

Sass imports don't treat CSS files as partials, so if we try to import vanilla CSS into a Sass file it is left as a regular CSS import, which won't work. This is a great case for reorganizing the stylesheets to be more modular (see: [FLUID-6552](https://issues.fluidproject.org/browse/FLUID-6552)) but I feel that it's outside the scope of this JIRA.

Reply:
Correction: the issue here is the file extension, so I can replace this with vanilla CSS and import it. However, this doesn't address the recommendation to move away from `@import` usage.

Reply:
Resolved in 2b06baf.